Readme.md

```diff
@@ -1,27 +1,36 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128565944/13.1.8%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E4942)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
 
-* [CorrectSolution.html](./CS/WebSite/CorrectSolution.html) (VB: [CorrectSolution.html](./VB/WebSite/CorrectSolution.html))
-* [Default.aspx](./CS/WebSite/Default.aspx) (VB: [Default.aspx](./VB/WebSite/Default.aspx))
-* [InvalidCase1.html](./CS/WebSite/InvalidCase1.html) (VB: [InvalidCase1.html](./VB/WebSite/InvalidCase1.html))
-* [InvalidCase2.html](./CS/WebSite/InvalidCase2.html) (VB: [InvalidCase2.html](./VB/WebSite/InvalidCase2.html))
-* [InvalidCase3.html](./CS/WebSite/InvalidCase3.html) (VB: [InvalidCase3.html](./VB/WebSite/InvalidCase3.html))
-* [InvalidCase4.html](./CS/WebSite/InvalidCase4.html) (VB: [InvalidCase4.html](./VB/WebSite/InvalidCase4.html))
-<!-- default file list end -->
-# Correct and incorrect examples of using height=100% in HTML
+# Relative height (100%) in DevExpress ASP.NET and DevExtreme controls
 <!-- run online -->
 **[[Run Online]](https://codecentral.devexpress.com/e4942/)**
 <!-- run online end -->
 
+DevExpress ASP.NET controls do not implement full screen feature (except for [ASPxSplitter](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxSplitter.FullscreenMode)). When a control’s height is set to 100%, it works incorrectly.
```

Comment:
- ` do not implement full screen feature` A/the feature? **full-screen feature**? And when you say do not implement, is this the same as support? 
- `it works incorrectly` What does this imply?

Reply:
```suggestion
DevExpress ASP.NET controls do not support the full screen feature out of the box (except for [ASPxSplitter](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxSplitter.FullscreenMode)). 

If you set a control’s height to 100%, it will not expand up to the page height, but on the contrary, it will collapse.

```

Reply:
DevExpress ASP.NET controls do not support the full-screen feature out of the box (except for [ASPxSplitter](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxSplitter.FullscreenMode)). 
If you set a control’s height to 100%, it will collapse instead of expanding to the page height.

Reply:
ok
```suggestion
DevExpress ASP.NET controls do not support the full-screen feature out of the box (except for [ASPxSplitter](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxSplitter.FullscreenMode)).
If you set a control’s height to 100%, it will collapse instead of expanding to the page height.
```